test/regress/regress1/quantifiers/issue3250-syg-inf-q.smt2

```diff
@@ -0,0 +1,14 @@
+(set-logic ALL)
```

Comment:
Shouldn't there be a `; COMMAND-LINE: --sygus-inference` for this test case, too?

Reply:
It doesn't work with sygus inference :)
I added it just since it works without.

Reply:
I've commented on this in the issue.

Reply:
Oh, I see :) Alright, let's leave it for now.